erpnext/agriculture/doctype/harvest/harvest.py

```diff
@@ -3,8 +3,62 @@
 # For license information, please see license.txt
 
 from __future__ import unicode_literals
-# import frappe
+import frappe
+import json
 from frappe.model.document import Document
 
 class Harvest(Document):
 	pass
+
+@frappe.whitelist()
+def create_stock_entry(harvest):
+	harvest = frappe.get_doc(json.loads(harvest))
```

Comment:
Why are we not using a `get_mapped_doc` instead? Might be easier to map and postprocess.

Reply:
we need to grab data from strain only linked field strain is reference so did that way